examples/reference-entity-model/flows/CurateCustomerJSON.flow.json

```diff
@@ -6,7 +6,114 @@
     },
     "2": {
       "stepId": "mapCustomersJSON-mapping"
+    },
+    "3": {
```

Comment:
@ryanjdew  Let me know if there's a better way to do a very simple match config on one property.

One thing I noticed is that the two customers that are merged together remain in the "Customer" collection. Is it up to the user to move them out of this collection? This results in the new, merged customer and those two original customers all appearing in search results, which doesn't seem desirable.

Reply:
There isn't a simpler way. Part of the reason we're looking to refactor the options.

The user could change the options to remove the Customer collection on archive. In the case of Customer, "sm-Customer-mastered" would be the collection to signify the distinct documents after mastering.